apps/web/src/scenes/NftDetailPage/NftDetailPageFallback.tsx

```diff
@@ -22,26 +55,50 @@ export default function NftDetailPageFallback() {
         <StyledThinSkeleton />
         <StyledButtonSkeleton />
         <StyledButtonSkeleton />
-      </VStack>
+      </StyledTextContainer>
     </StyledFullPageLoader>
   );
 }
 
 const StyledFullPageLoader = styled.div`
   display: flex;
-  gap: 60px;
   height: 100vh;
   width: 100%;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+
+  @media only screen and (max-width: ${size.tablet}px) {
```

Comment:
lets use mobile first responsive styling for consistency, that way it's easier to reason about when the style gets applied

so each `@media only...` will apply styles for the defined screen size or larger